bacula/baculum/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:20.04
+
+MAINTAINER ops@marfeel.com
+
+RUN apt-get update && apt-get install -y gnupg wget joe ca-certificates
```

Comment:
Wether `joe` leaves or not, this could also be improved cosmetically to improve maintainability

```dockerfile
RUN apt-get update && apt-get install -y \
    gnupg \
    wget \
    joe \
    ca-certificates
```

Reply:
Also, just realized it is missing the --no-install-recommends flag, just updated the comment above.